readme.md

```diff
@@ -5,9 +5,9 @@ Seen these cool projects on github? Pain to download? This program is for you!
 
 Installation
 ------------
-If only there were some easy way to download this program from github...
+The easiest way to install this project on your computercraft computer is by running:
 
-Copy both files in `apis/` to `/apis/`, and put `programs/github` wherever you want it.
+    pastebin run p8PJVxC4
```

Comment:
I like the idea of doing that in an install file... simply for simplicity sake, will change and push.